database.sql

```diff
@@ -3,24 +3,7 @@ USE groupc_kainos_recruitment_JenniferC;
 
 CREATE TABLE IF NOT EXISTS job_roles (
 id smallint PRIMARY KEY AUTO_INCREMENT,
-name varchar(50),
-job_description varchar(500)
+name varchar(50) NOT NULL,
+job_description varchar(500) NOT NULL,
+capability varchar(50)
```

Comment:
Capability is from a list of specific values, what about using the enum data type?